eth/filters/filter_test.go

```diff
@@ -138,6 +138,7 @@ func TestFilters(t *testing.T) {
 				},
 			}
 			gen.AddUncheckedReceipt(receipt)
+			gen.AddUncheckedTx(types.NewTransaction(1, common.HexToAddress("0x1"), big.NewInt(1), 1, big.NewInt(1), nil))
```

Comment:
Why do you add it?

Reply:
I added this because `core.BlockGen` performs checks on receipts that would not be valid during this test. I saw that we already had a similar method for transactions (`AddUncheckedTx`), so I figured it was fair game to do the same for a receipt.